### ISSUE TITLE
Intelisense stopped working

### ISSUE THREAD

**Issue by sanket-bhalerao** [Microsoft/vscode#49362]
Issue Type: <b>Bug</b>

Intellisense has stopped working after latest update.

VS Code version: Code - Insiders 1.24.0-insider (d29620f2ba89327f51bb6fb2f05dfdd65e2fcafe, 2018-05-07T08:55:27.483Z)
OS version: Windows_NT x64 10.0.15063

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz (4 x 3292)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|7.96GB (2.48GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
ng-template|Ang|0.1.9
jshint|dba|0.10.18
vscode-eslint|dba|1.4.8
gitlens|eam|8.2.4
EditorConfig|Edi|0.12.1
tslint|eg2|1.0.28
python|ms-|2018.4.0
PowerShell|ms-|1.7.0
typescript-javascript-grammar|ms-|0.0.41
debugger-for-chrome|msj|4.4.3
vscode-react-native|vsm|0.6.9

</details>

**Comment by marvinhagemeister** [comment]
Debug output from developer console:

```
 Cannot read property 'getConfiguration' of undefined: TypeError: Cannot read property 'getConfiguration' of undefined
    at FileConfigurationManager.getPreferences (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:106:45)
    at FileConfigurationManager.getFileOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:73:31)
    at FileConfigurationManager.ensureConfigurationOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:59:37)
    at FileConfigurationManager.ensureConfigurationForDocument (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:49:25)
    at TypeScriptCompletionItemProvider.provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/completionItemProvider.js:215:45)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:661:233
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:96:887
    at new n.Class.derive._oncancel (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:76:776)
    at Object.t.asWinJsPromise (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:96:850)
    at e.provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:661:187)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:673:315
    at e._withAdapter (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:666:579)
    at e.$provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:673:277)
    at e._doInvokeHandler (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:602:894)
    at e._invokeHandler (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:602:612)
    at e._receiveRequest (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:601:682)
    at e._receiveOneMessage (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:601:451)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:600:384
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:603:463
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:88:432
    at e.fire (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:89:764)
    at Socket.<anonymous> (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:146:1016)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:560:20)

log.ts:171   ERR Cannot read property 'getConfiguration' of undefined: TypeError: Cannot read property 'getConfiguration' of undefined
    at FileConfigurationManager.getPreferences (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:106:45)
    at FileConfigurationManager.getFileOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:73:31)
    at FileConfigurationManager.ensureConfigurationOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:59:37)
    at FileConfigurationManager.ensureConfigurationForDocument (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:49:25)
    at TypeScriptQuickFixProvider.provideCodeActions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/quickFixProvider.js:143:51)
    at <anonymous>

log.ts:171   ERR Cannot read property 'getConfiguration' of undefined: TypeError: Cannot read property 'getConfiguration' of undefined
    at FileConfigurationManager.getPreferences (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:106:45)
    at FileConfigurationManager.getFileOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:73:31)
    at FileConfigurationManager.ensureConfigurationOptions (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:59:37)
    at FileConfigurationManager.ensureConfigurationForDocument (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/fileConfigurationManager.js:49:25)
    at TypeScriptCompletionItemProvider.provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/extensions/typescript-language-features/out/features/completionItemProvider.js:215:45)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:661:233
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:96:887
    at new n.Class.derive._oncancel (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:76:776)
    at Object.t.asWinJsPromise (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:96:850)
    at e.provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:661:187)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:673:315
    at e._withAdapter (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:666:579)
    at e.$provideCompletionItems (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:673:277)
    at e._doInvokeHandler (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:602:894)
    at e._invokeHandler (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:602:612)
    at e._receiveRequest (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:601:682)
    at e._receiveOneMessage (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:601:451)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:600:384
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:603:463
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:88:432
    at e.fire (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:89:764)
    at Socket.<anonymous> (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/node/extensionHostProcess.js:146:1016)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:560:20)
```

**Comment by marvinhagemeister** [comment]
Seems to be fixed in 8a07c8d935998d1b1153dde0c5b337189d35cea0

**Comment by mjbvz** [comment]
This
